### ISSUE TITLE
Can't download anything?

### ISSUE THREAD

**Issue by Zaybomb** [Special-Niewbie/Nintendont-CrisMMMod#12]
[Here is a video](https://photos.app.goo.gl/uMMF2UWNBm7vxNBM9) 

So basically when ever i try to download something it gives me this error.

**Comment by Special-Niewbie** [comment]
Hi @Zaybomb, strange..... Are using it on Wii U by vWii, right? Let me know please because I need to replicate your error to understand where is your problem, and another thing, have you the SD Card locked?

**Comment by Zaybomb** [comment]
Wii u by vwii and my sd card is not locked.

**Comment by Special-Niewbie** [comment]
> Wii u by vwii and my sd card is not locked.

How did you called the folder where it's placed the .dol file? Later I will check on one of my Wii U......

**Comment by Zaybomb** [comment]
Nintendont CMMM

**Comment by Special-Niewbie** [comment]
> Nintendont CMMM

This could be the problem...... because cannot find the folder.....Rename it just:  Nintendont

And please, let me know 😉

PS: after that you should download anything, except the Update, because the Update there is a problem that I need to find the way to fix it ..

**Comment by Zaybomb** [comment]
Ok just a sec

**Comment by Zaybomb** [comment]
No dice (it's still not working, same error)

**Comment by Zaybomb** [comment]
Hmmmm, this is weird because my vwii is connected to the internet and can download covers for usb loaders

**Comment by Special-Niewbie** [comment]
> Hmmmm, this is weird because my vwii is connected to the internet and can download covers for usb loaders

Let me check later, and I will let you know, but I remember that wasn't any problem like that....I will let you know

**Comment by Zaybomb** [comment]
K

**Comment by Zaybomb** [comment]
Would sd card size matter? Mine is 64 gig

**Comment by Special-Niewbie** [comment]
@Zaybomb , yes, I confirm you that it starts to gives this problem thanks to point it out, I need to check why it happen. Thanks a lot.

The SD size it's not matter as soon as you have enough space in it. Anyway, it seems it's not your problem, I have to check why....

**Comment by Zaybomb** [comment]
I did not think it was the card size because i can play the games just fine. Weird that it is happening to you too...

**Comment by Special-Niewbie** [comment]
Because the problem could be in the server where are located the files when need to upgrade.....the app cannot, could make this problem, but for sure it's somewhere , last time when I tried was stable in both console Wii and Wii U... let's see

**Comment by Zaybomb** [comment]
Ya i was just gonna say that my wii is getting the same error. Are you self-hosting the files or is it on some one else's servers? Also, do you have a manual way to download them?

**Comment by Special-Niewbie** [comment]
It's self-virtual hosting, you can download any file of this project openly in my main page of Nintendont-CrisMMMod, navigate into the `nintendont` folder in this GitHub and you will find the reference files that you need for now.....
